core/config/natAssay.js

```diff
@@ -0,0 +1,14 @@
+import harden from '@agoric/harden';
+
+import natStrategy from './natStrategy';
+import { makeAssayMaker } from '../assay';
+//
+// The default assay makes the default kind of amount.  The default
+// kind of amount is a labeled natural number describing a quantity of
+// fungible erights. The label describes what kinds of rights these
+// are. This is a form of labeled unit, as in unit typing.
```

Comment:
Would you mind adding a line pointing out that we use natural numbers because floats have such problematic semantics? The strategy is to choose a small enough unit that it never has to deal with fractional parts. I didn't get that this was the point until I said something and got a reaction of disbelief that I had to interpret.